packages/x-date-pickers/src/internals/hooks/useField/useField.ts

```diff
@@ -17,13 +17,15 @@ import { useFieldState } from './useFieldState';
 import { useFieldCharacterEditing } from './useFieldCharacterEditing';
 import { getActiveElement } from '../../utils/utils';
 import { FieldSection } from '../../../models';
+import { TimeValidationProps } from '../../models/validation';
 
 export const useField = <
   TValue,
   TDate,
   TSection extends FieldSection,
   TForwardedProps extends UseFieldForwardedProps,
-  TInternalProps extends UseFieldInternalProps<any, any, any, any>,
+  TInternalProps extends UseFieldInternalProps<any, any, any, any> &
+    Partial<TimeValidationProps<TDate>>,
```

Comment:
The partial is not needed here


I'm not a fan of adding all the time validation to `useField` (which is supposed to be agnostic, how about we just go for `& { minuteStep?: number }` ?

Reply:
Fine, then we will be able to add more properties in the future is needed